src/Form/Type/VichImageType.php

```diff
@@ -43,6 +43,9 @@ public function configureOptions(OptionsResolver $resolver): void
             'image_uri' => true,
             'imagine_pattern' => null,
             'storage_resolve_method' => self::STORAGE_RESOLVE_URI,
+            'attr' => [
+                'accept' => 'image/png,image/jpeg,image/gif,image/webp',
```

Comment:
`'image/*'` is probably better (more synthetic and open to possible other formats)

Reply:
> `'image/*'` is probably better (more synthetic and open to possible other formats)

Agreed, and done